web/tests/settings_bots.test.js

```diff
@@ -102,45 +92,6 @@ test("generate_botserverrc_content", () => {
     assert.equal(content, expected);
 });
 
-test("test tab clicks", () => {
-    settings_bots.set_up();
-
-    function click_on_tab($tab_elem) {
-        $tab_elem.trigger("click");
-    }
-
-    const tabs = {
-        $active: $("#bots_lists_navbar .active-bots-tab"),
-        $inactive: $("#bots_lists_navbar .inactive-bots-tab"),
-    };
-
-    $("#bots_lists_navbar .active").removeClass = (cls) => {
-        assert.equal(cls, "active");
-        for (const $tab of Object.values(tabs)) {
-            $tab.removeClass("active");
-        }
-    };
-
-    const forms = {
-        $active: $("#active_bots_list"),
-        $inactive: $("#inactive_bots_list"),
-    };
-
-    click_on_tab(tabs.$active);
-    assert.ok(tabs.$active.hasClass("active"));
-    assert.ok(!tabs.$inactive.hasClass("active"));
-
-    assert.ok(forms.$active.visible());
-    assert.ok(!forms.$inactive.visible());
-
-    click_on_tab(tabs.$inactive);
-    assert.ok(!tabs.$active.hasClass("active"));
-    assert.ok(tabs.$inactive.hasClass("active"));
-
-    assert.ok(!forms.$active.visible());
-    assert.ok(forms.$inactive.visible());
-});
```

Comment:
Removed this test, since it would have needed mocked functions and elements for code in `components.toggle` and we already test the code in `components.ts`.